flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/window/slicing/SliceAssigners.java

```diff
@@ -552,7 +553,9 @@ protected AbstractSliceAssigner(int rowtimeIndex, ZoneId shiftTimeZone) {
         public final long assignSliceEnd(RowData element, ClockService clock) {
             final long timestamp;
             if (rowtimeIndex >= 0) {
-                timestamp = toUtcTimestampMills(element.getLong(rowtimeIndex), shiftTimeZone);
```

Comment:
we need to fix other getLong in the assigners.

Reply:
Working on it. I will also check for other operators

Reply:
@wenlong88 I have fixed all problems relating to Window TVF. For other similar occurrences, I suggest that we open a separate issue.